samza/src/test/scala/com/metamx/tranquility/test/SamzaDruidTest.scala

```diff
@@ -114,6 +118,70 @@ class SamzaDruidTestInputSystemFactory extends SystemFactory
   }
 }
 
+class SamzaDruidCoordinatorSystemFactory extends SystemFactory
```

Comment:
New samza uses `System` for storing configs. This is test `SystemFactory` which allow us test new samza without starting test kafka broker.